.checkpatch.conf

```diff
@@ -8,5 +8,7 @@
 --ignore=BIT_MACRO
 --ignore=PREFER_PRINTF
 --ignore=PREFER_SCANF
+--ignore=VOLATILE
```

Comment:
I tried that and I don't think it did anything useful. The problem I've always had with git is that eventually clones get too tangled to do anything useful with them as you try to keep them in sync with the master repo with fetches and merges. I usually just use a fresh clone to start over. I may need to do that here. Will see what Maxim suggests when he gets back.